plugins/Jobs.cpp

```diff
@@ -253,18 +253,22 @@ bool BedrockPlugin_Jobs::peekCommand(SQLite& db, BedrockCommand& command) {
             // Verify unique, but only do so when creating a single job using CreateJob
             if (SIEquals(request.methodLine, "CreateJob") && SContains(job, "unique") && job["unique"] == "true") {
                 SQResult result;
-                SINFO("Unique flag was passed, checking existing job with name " << job["name"]);
+                SINFO("Unique flag was passed, checking existing job with name " << job["name"] << ", mocked? "
+                      << (command.request.isSet("mockRequest") ? "true" : "false"));
+                string operation = command.request.isSet("mockRequest") ? "IS NOT" : "IS";
                 if (!db.read("SELECT jobID, data "
                              "FROM jobs "
-                             "WHERE name=" + SQ(job["name"]) + ";",
+                             "WHERE name=" + SQ(job["name"]) +
+                             "  AND JSON_EXTRACT(data, '$.mockRequest') " + operation + " NULL;",
```

Comment:
I think we should time this to be sure we are not introducing a performance regression.